design-library/src/components/BccPagination/BccPagination.css

```diff
@@ -0,0 +1,64 @@
+@layer components {
+
+    .bcc-pagination {
+        @apply flex flex-row items-center justify-between
+    }
+
+    .bcc-pagination-right {
+        @apply flex flex-row items-center justify-between
+    }
+
+    .bcc-pagination-left {
+        @apply flex flex-row-reverse items-center justify-between
+    }
+
+    .bcc-pagination-center {
+        @apply flex flex-row items-center justify-evenly
+    }
+
+    .bcc-pagination-rowsperpage-container {
+        @apply inline-flex flex-row items-center gap-2
+    }
+
+    .bcc-pagination-rowsperpage {
+        @apply w-20 text-caption
+    }
+
+    .bcc-pagination-button-container {
+        @apply inline-flex items-center
+    }
+
+
+    .bcc-pagination-button-container button {
+        @apply select-none inline-flex items-center tracking-wide border border-on-primary justify-center active:shadow-inner focus-visible:outline focus-visible:outline-2 focus-visible:outline-emphasis focus-visible:outline-offset-2;
+        @apply cursor-pointer disabled:opacity-40 disabled:cursor-not-allowed disabled:pointer-events-none;
+        
+        /* Default base size */
+        @apply text-base leading-5 py-2.5 gap-x-2 h-10;
+    }
+
+    .bcc-pagination-button-container button:hover {
+        @apply bg-secondary-hover text-interactive-hover
+    }
+
+    .bcc-pagination-button-selected {
+        @apply px-3.5 bg-secondary font-semibold text-interactive;
+    }
+ 
+    .bcc-pagination-button-unselected {
+        @apply px-3.5 bg-button-secondary text-secondary;
+    }
+
+    .bcc-pagination-button-left {
+        @apply rounded-l-lg px-3.5 text-interactive;
+    }
+
+    .bcc-pagination-button-right {
+        @apply rounded-r-lg px-3.5 text-interactive;
+    }
```

Comment:
Same comment on consolidating these in one `bcc-pagination-button` (or just call them `bcc-pagination-arrow`?) and then a modifier class that only changes the rounding.

Reply:
when I fixed the double border, the right arrow needed different tailwind code and its so little shared code that I don't think its worth having a separate class